config/initializers/sidekiq.rb

```diff
@@ -23,6 +23,16 @@
   end
 end
 
+if ENV.key?("REDIS_URI")
```

Comment:
`REDIS_URI` is set in the GitHub Codespaces environment variable settings. As per the the docker compose it's `redis://redis`.